nav2_simple_commander/nav2_simple_commander/costmap_2d.py

```diff
@@ -71,3 +71,72 @@ def getGlobalFrameID(self):
     def getCostmapTimestamp(self):
         """Get costmap timestamp."""
         return self.costmap_timestamp
+
+    def getCost(self, _index: int = None, _mx: int = None, _my: int = None) -> np.int8:
```

Comment:
Why are these starting with underscores? For all functions, I don't think that's proper

Reply:
Okay, I will remove all the underscore

Reply:
remove all the underscores